internal/shim-sev/src/sse.rs

```diff
@@ -7,8 +7,7 @@ use crate::snp::cpuid_count;
 
 use core::arch::asm;
 
-use x86_64::registers::xcontrol::{XCr0, XCr0Flags};
-use xsave::MxCsr;
+use x86_64::registers::{xcontrol::{XCr0, XCr0Flags}, mxcsr::MxCsr};
```

Comment:
You should run `cargo fmt` before submitting a PR. Or better automate your editor to do it on save for you.

```suggestion
use x86_64::registers::{
    mxcsr::MxCsr,
    xcontrol::{XCr0, XCr0Flags},
}
```

Reply:
Sure, I will take care of this.